src/pages/workspace/NewWorkspacePage.js

```diff
@@ -0,0 +1,144 @@
+     */
+    createMenuItems(openPicker) {
+        const menuItems = [
+            {
+                icon: Upload,
+                text: this.props.translate('workspace.new.uploadPhoto'),
+                onSelected: () => {
+                    openPicker({
+                        onPicked: () => {
+                            // TODO: connect with setWorkspaceAvatar function
+                        },
+                    });
+                },
+            },
+        ];
+
+        // TODO: Add option to remove avatar if the user doesn't like the one they chose.
+
+        return menuItems;
+    }
+
+    render() {
+        const {translate} = this.props;
+
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={translate('workspace.new.welcome')}
+                    onCloseButtonPress={Navigation.dismissModal}
+                />
+
+                <View style={[styles.pageWrapper, styles.flex1]}>
+                    {/* TODO: replace this with the Avatar component once we connect it with the backend */}
+                    <WorkspaceDefaultAvatar height={100} width={100} />
+
+                    <AttachmentPicker>
+                        {({openPicker}) => (
+                            <>
+                                <Button
+                                    style={[styles.alignSelfCenter, styles.mt3]}
+                                    onPress={() => this.setState({isEditPhotoMenuVisible: true})}
+                                    ContentComponent={() => (
+                                        <View style={[styles.flexRow]}>
+                                            <Icon src={DownArrow} />
+                                            <View style={styles.justifyContentCenter}>
+                                                <Text style={[styles.headerText, styles.ml2]}>
+                                                    {translate('workspace.new.editPhoto')}
+                                                </Text>
+                                            </View>
+                                        </View>
+                                    )}
+                                />
+                                <CreateMenu
+                                    isVisible={this.state.isEditPhotoMenuVisible}
+                                    onClose={() => this.setState({isEditPhotoMenuVisible: false})}
+                                    onItemSelected={() => this.setState({isEditPhotoMenuVisible: false})}
+                                    menuItems={this.createMenuItems(openPicker)}
+                                    anchorPosition={styles.createMenuPositionProfile}
+                                    animationIn="fadeInRight"
+                                    animationOut="fadeOutRight"
+                                />
+                            </>
+                        )}
+                    </AttachmentPicker>
```

Comment:
NAB, this _might_ be a case where we can convert this pattern into a slightly more complex component. But we can perhaps wait until it is used a third time? Just something I thought about since we have mostly the same UI pattern used in the `ProfilePage.js`